src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PAYMENT = 40;
+  const BIGGER_DISCONT = 7;
+  const SMALLER_DISCONT = 3;
+  let sum = days * PAYMENT;
```

Comment:
Prefer using 'const' over 'let' for the 'sum' variable since it's not being reassigned but only modified, which is allowed for 'const' variables.

Reply:
when adding requesting changes can't pass linter